src/typings/jwt.typing.ts

```diff
@@ -0,0 +1,7 @@
+export type JWT = {
```

Comment:
I suggest changing to `Jwt` since eveyrone use in this way:
```suggestion
export type Jwt = {
```

Reply:
Resolved on da52113582ec5b059a145f6eff3d20158850aa5b.